apps/sms/style/sms.css

```diff
@@ -933,16 +932,24 @@ body[role="application"] form[role="dialog"][data-type="action"] > header:first-
 }
 
 /* Phone details label styles */
-.phone-type,
+.carrier-information {
+  display: flex;
+}
+
+.phone-type, .phone-number, .phone-carrier {
+  max-width: 60%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  flex: 0 1 auto;
+}
+
 .phone-type-separator,
-.phone-carrier,
 .phone-carrier-separator {
   display: none;
+  flex: 0 0 0.3em;
```

Comment:
Here is a new proposal to make it work like Fang would like:

```css
flex: none;
padding-right: 0.3em;
```

This seems to work perfect for me. What do you think?

The idea is that the flex element takes the "normal" space it should take (content + padding), and then never grow/shrink.